packages/shopping/src/__tests__/e2e/integration/cart.spec.js

```diff
@@ -0,0 +1,64 @@
+context('Cart', () => {
+  before(() => cy.visit('http://localhost:3000/shoppy.html'));
+
+  it('should checkout all items from Cart', () => {
+    cy.get('#logIn')
+      .click()
+      .then(() => cy.get('#logInButton').click());
+
+    cy.get('#shoppingCartLink')
+      .click()
+      .then(() =>
+        cy
+          .get('.list-group')
+          .find('li')
+          .get('input[name="cartItem"]')
+          .should('be.visible')
+          .check(),
+      );
+    cy.get('#checkOutButton').click();
+  });
+
+  it('should add items to Cart', () => {
+    cy.get('#products')
+      .find('.card')
+      .first()
+      .get('.action-buttons')
+      .contains('Details')
+      .click();
+
+    addToCart();
+  });
+
+  it('should remove all items from Cart', () => {
+    cy.get('#logInButton').click({force: true});
+    cy.get('#shoppingCartLink')
```

Comment:
Similarly here, the modal from the previous test is still open so this test fails:

<img width="1195" alt="Screen Shot 2020-10-21 at 9 24 25 AM" src="https://user-images.githubusercontent.com/42985749/96725747-37a0ee00-137f-11eb-9132-fc864693732b.png">

Reply:
Will apply same pattern, as above. Interesting enough on my machine they're all working

Reply:
> Will apply same pattern, as above. Interesting enough on my machine they're all working

Worked locally for me too.  We're all learning. Great work btw.

Reply:
Did you run only this test or all the tests? By itself, the test runs fine, but the failure could just be from my machine.

> Great work btw.

+1

Reply:
I ran all the tests and it went smoothly at the time of review. For the best that you had an issue as it prompted further refinement. 👍